tools/importer-rest-api-specs/components/dataapigeneratoryaml/templater_package_definition.go

```diff
@@ -0,0 +1,66 @@
+package dataapigeneratoryaml
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/go-yaml/yaml"
+	"github.com/hashicorp/pandora/tools/importer-rest-api-specs/models"
+)
+
+type Package struct {
+	Name       string   `yaml:"Name"`
+	Operations []string `yaml:"Operations"`
+	Constants  []string `yaml:"Constants"`
+	Models     []string `yaml:"Models"`
+}
+
+func codeForPackageDefinition(resourceName string, input models.AzureApiResource) ([]byte, error) {
+	operationNames := make([]string, 0)
+	for operation := range input.Operations {
+		operationNames = append(operationNames, operation)
+	}
+	sort.Strings(operationNames)
+
+	operations := make([]string, 0)
+
+	for _, operationName := range operationNames {
+		operations = append(operations, fmt.Sprintf("%sOperation", operationName))
+	}
```

Comment:
we don't need to suffix these now, since we're outputting these into different files/subsections and not using reflection to find these - meaning we can just use `operationNames` here?

```suggestion
	operationNames := make([]string, 0)
	for operation := range input.Operations {
		operationNames = append(operationNames, operation)
	}
	sort.Strings(operationNames)
```